src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerTest.java

```diff
@@ -108,9 +80,176 @@ private static HearingEntity setupCase3(CourthouseEntity swanseaCourthouse, Cour
         case1.setProsecutors(List.of("Prosecutor00000031", "Prosecutor00000032"));
 
         var hearingForCase1 = aHearingForCaseInRoom(case1, swanseaCourtroom1);
-        hearingForCase1.setJudges(List.of("{Judge1}"));
+        hearingForCase1.setJudges(List.of("{Judge3}"));
         hearingForCase1.setHearingDate(LocalDate.parse(HEARING_DATE));
         hearingForCase1.setScheduledStartTime(LocalTime.parse("11:00"));
         return hearingForCase1;
     }
+
+    private static HearingEntity setupCase4(CourthouseEntity swanseaCourthouse, CourtroomEntity swanseaCourtroom1) {
```

Comment:
Might be helpful to add a comment to say what type of case situation this is setting up

Reply:
ACCEPT